.github/workflows/daily-csv-upload.yaml

```diff
@@ -0,0 +1,69 @@
+name: Upload current-full.csv and current-federal.csv
+run-name: Upload current-full.csv and current-federal.csv for branch ${{ github.head_ref }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Which environment do you wish to load reports for?
+        options:
+          - stable
+          - staging
+          - development
+          - ky
+          - es
+          - nl
+          - rh
+          - za
+          - gd
+          - rb
+          - ko
+          - ab
+          - bl
+          - rjm
+          - dk
+  # TODO - uncomment after #1403 is finished
+  #schedule:
+    # Runs every day at 5 AM UTC.
+    # - cron: "0 5 * * *"
+
+jobs:
+  variables: 
+    outputs:
+      environment: ${{ steps.var.outputs.environment }}
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Setting global variables
+        uses: actions/github-script@v6
+        id: var
+        with:
+          script: |
+            const environment = (github && github.event && github.event.inputs) ? github.event.inputs.environment : 'za';
+            core.setOutput('environment', environment);
+
+  upload-reports:
+    runs-on: ubuntu-latest
+    needs: [variables]
+    env:
+      CF_USERNAME: CF_{{ needs.variables.outputs.environment }}_USERNAME
+      CF_PASSWORD: CF_{{ needs.variables.outputs.environment }}_PASSWORD
+    steps:
+      - name: Generate current-federal.csv
+        uses: 18f/cg-deploy-action@main
```

Comment:
I believe you'll need to revise the script used per @neilmb's PR. Again on line 62.

Reply:
Thanks Rachid, I assume to "cloud-gov/cg-cli-tools@main"?